app/models/like.rb

```diff
@@ -1,4 +1,15 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  validates :user_id, presence: true
+  validates :post_id, presence: true
+
+  after_create :update_post_likes_counter
+
+  private
+
+  def update_post_likes_counter
+    post.update_likes_counter
+  end
```

Comment:
- [Optional] Instead of having two method to achieved this objective, we could have call the increment method as mentioned above on a post `Hints`

```
 def update_post_likes_counter
     post.increment!(:likes_counter)
 end
```

- With this in place, we can now get rid of the `update_likes_counter` method in the Post model. This makes our codebase simple :+1: